app/bundles/LeadBundle/Segment/Decorator/DateDecorator.php

```diff
@@ -11,21 +11,10 @@
 
 namespace Mautic\LeadBundle\Segment\Decorator;
 
-use Mautic\CoreBundle\Helper\CoreParametersHelper;
 use Mautic\LeadBundle\Segment\ContactSegmentFilterCrate;
-use Mautic\LeadBundle\Segment\ContactSegmentFilterOperator;
-use Mautic\LeadBundle\Services\ContactSegmentFilterDictionary;
 
 class DateDecorator extends CustomMappedDecorator
 {
-    public function __construct(
-        ContactSegmentFilterOperator $contactSegmentFilterOperator,
```

Comment:
I see these arguments in config.php

Reply:
And that's fine. The parent constructor will take over.

Reply:
oh ...